src/person.rb

```diff
@@ -13,10 +13,15 @@ def initialize(age, name: 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
+  end
+
+  def add_rental(date, person)
+    Rental.new(date, self, person)
   end
```

Comment:
- [ ] Please kindly take note that the `self` keyword typically refers to the current instance of the class where the code is located (in this case `Person`); I think you will be more accurate to pass the `book` parameter as part of the arguement as stated in the requirement and remove `person`. ℹ️ 

<img width="638" alt="project requirement" src="https://github.com/Kaiserabbas/school_library/assets/103386183/5919b154-6a2e-43c7-a7ee-d6359794586c">

- [ ] Kindly ensure that   *Getters and Setters for @rentals is present, using getters and setters in your classes provides a level of control over attribute access. ℹ️